README.md

```diff
@@ -103,26 +103,40 @@ you'll get an error. You can fix this by either logging in (via `docker login`)
 
 Building
 -
-To build the manual you will need go-md2man.
+To build the `skopeo` binary you need at least Go 1.5 because it uses the latest `GO15VENDOREXPERIMENT` flag.
+
+### Building in a container
+If your system has a `docker` command and can run Linux containers,
 ```sh
-$ sudo apt-get install go-md2man
+$ make binary # Or (make all) to also build documentation, see below.
```

Comment:
I’m afraid I have no idea; random googling suggests that this is a problem when cross-compiling from Linux to macOS.
> ```
> CGO_ENABLED=1 GOARCH=amd64 GOOS=darwin go build …
> ```

How did the `CGO_ENABLED=1 GOARCH=amd64 GOOS=darwin` get there? I can’t see anything like that in the `Makefile`. Is `go` doing some magic autodetection, or is that a local modification?